java/impl/aws/src/main/java/co/worklytics/psoxy/aws/request/APIGatewayV1ProxyEventRequestAdapter.java

```diff
@@ -83,6 +83,16 @@ public String prettyPrint() {
         return event.toString();
     }
 
+    @Override
+    public Optional<String> getClientIp() {
+        //this correct?
+        // standard: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Forwarded-For
+        String ip = Optional.ofNullable(event.getHeaders().get("X-Forwarded-For"))
```

Comment:
Can we have `X-Forwarded-For` as a public final static var somewhere? As this is used in aws/gcp stuff